x-pack/plugins/security_solution/server/endpoint/routes/trusted_apps/index.ts

```diff
@@ -5,95 +5,96 @@
  * 2.0.
  */
 
-import {
-  DeleteTrustedAppsRequestSchema,
-  GetOneTrustedAppRequestSchema,
-  GetTrustedAppsRequestSchema,
-  PostTrustedAppCreateRequestSchema,
-  PutTrustedAppUpdateRequestSchema,
-  GetTrustedAppsSummaryRequestSchema,
-} from '../../../../common/endpoint/schema/trusted_apps';
-import {
-  TRUSTED_APPS_CREATE_API,
-  TRUSTED_APPS_DELETE_API,
-  TRUSTED_APPS_GET_API,
-  TRUSTED_APPS_LIST_API,
-  TRUSTED_APPS_UPDATE_API,
-  TRUSTED_APPS_SUMMARY_API,
-} from '../../../../common/endpoint/constants';
-
-import {
-  getTrustedAppsCreateRouteHandler,
-  getTrustedAppsDeleteRouteHandler,
-  getTrustedAppsGetOneHandler,
-  getTrustedAppsListRouteHandler,
-  getTrustedAppsSummaryRouteHandler,
-  getTrustedAppsUpdateRouteHandler,
-} from './handlers';
+// import {
```

Comment:
I know its ugly. But its temporary and is the least impactful way to ensure the APIs are not available in 8.0

Reply:
😨

Reply:
What's the difference between removing vs commenting the code?

Reply:
@paul-tavares you can always revert a commit or reference if necessary. better to remove the code than commenting.